packages/ra-supabase-ui-materialui/src/LoginPage.tsx

```diff
@@ -72,7 +72,6 @@ import {
  * );
  */
 export const LoginPage = (props: LoginPageProps) => {
-    useRedirectIfAuthenticated();
```

Comment:
why did you remove this? :thinking:

Reply:
Because this leads to an infinite redirection. Also, I think it doesn't make sense to redirect users if they want access this page: they might want to log in with other credentials.

Reply:
Kept, and added to the the changelog